encrypt_test.go

```diff
@@ -186,3 +187,62 @@ func Test_EncryptManager_AES256_CFB(t *testing.T) {
 		})
 	}
 }
+
+func Test_EncryptManager(t *testing.T) {
+
+	// open a sample file
+	original, err := ioutil.ReadFile("sample_data")
```

Comment:
why not `original := []byte("Test data to be encrypted")`?

Reply:
Done.